packages/cloud-cognitive/src/components/BreadcrumbWithOverflow/BreadcrumbWithOverflow.js

```diff
@@ -310,17 +309,15 @@ export let BreadcrumbWithOverflow = ({
         >
           {backItem?.href && (backItem?.label || backItem?.title) && (
             <BreadcrumbItem className={cx(`${blockClass}__breadcrumb-back`)}>
-              <Link
+              <IconButton
```

Comment:
This results in an additional 4px of spacing (+6 above and -2px below) over V10.

Applying the following would address this change

```scss
.c4p--breadcrumb-with-overflow__space {
  margin-top: -6px;
 }
```